deploifai/training_server/create.py

```diff
@@ -0,0 +1,382 @@
+import click
+from PyInquirer import prompt
+
+from deploifai.context import pass_deploifai_context_obj, DeploifaiContextObj, is_authenticated, project_found
+from deploifai.utilities.version_comparision import compare, multi_compare
+from deploifai.utilities.python_supported import supported
+
+framework_options = ["tensorflow", "pytorch", "no_framework"]
+server_options = ["SMALL", "MEDIUM", "LARGE"]
+
+
+@click.command()
+@click.argument("name")
+@click.option("--framework", '-fw', type=str,
+              help='ML framework to be installed in the server: {}'.format(framework_options))
+@click.option("--framework-version", '-fwv', type=str, help="ML framework version to be installed")
+@click.option("--python-version", '-py', type=str, help="Python version to be installed (works with selected ML framework)")
```

Comment:
Add `--no-framework` option that defaults to false

Reply:
added